package.json

```diff
@@ -18,24 +18,24 @@
     "console"
   ],
   "dependencies": {
-    "pty.js": "latest",
-    "jquery": "latest",
-    "lodash": "latest",
-    "node-ansiparser": "latest",
-    "react": "latest"
+    "pty.js": "^0.2",
+    "jquery": "^2.1",
+    "lodash": "^3.7",
+    "node-ansiparser": "^1.0",
+    "react": "^0.13"
   },
   "devDependencies": {
-    "electron-prebuilt": "latest",
-    "gulp": "latest",
+    "electron-prebuilt": "^0.24",
+    "gulp": "^3.8",
     "gulp-react": "^3.0.1",
-    "gulp-sass": "^1.3.3",
+    "gulp-sass": "^2.0.0-alpha.1",
     "gulp-typescript": "^2.6.0",
-    "node-sass": "latest",
-    "react-tools": "latest",
-    "typescript": "latest"
+    "node-sass": "^3.0.0-beta.7",
+    "react": "^0.13",
```

Comment:
didn't notice that react-tools has changed to just `react`
Now it's in both deps lists